sdk/digitaltwins/digital-twins-core/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release History
 
+## 1.0.0 (2020-10-30)
+
+- The is the GA release containing the following changes:
+  - Expose ifMatch, ifNoneMatch optional parameters for upsertDigitalTwin and upsert Relationship APIs
```

Comment:
Other packages (e.g. AppConfig) use `onlyIfChanged` and `onlyIfUnchanged` boolean properties when updating resources to provide semantic meaning to eTag operations, as we don't generally expect developers to be experts in HTTP conditional requests.

Reply:
We are agreed on using "ifMatch" and "ifNoneMatch" across the languages

Reply:
Yeah it seems like this is the way to go if we only have the id/etag